cheddar/cheddar-metrics-intercom/src/main/java/com/clicktravel/cheddar/metrics/intercom/IntercomMetricCollector.java

```diff
@@ -34,15 +33,17 @@
 
 public class IntercomMetricCollector implements MetricCollector {
     private final Logger logger = LoggerFactory.getLogger(getClass());
-    private MetricCustomAttributeToIntercomCustomAttributeMapper customAttributeMapper;
+    private MetricCustomAttributeToIntercomCustomAttributeMapper metricToIntercomCustomAttributeMapper;
+    private final IntercomCustomAttributeToMetricCustomAttributeMapper intercomToMetricCustomAttributeMapper;
```

Comment:
It looks like it is to allow the mapper to be mocked out in unit tests, which does make sense.  I think we should follow the same approach here.